libs/category/src/reducers/category/category.reducer.ts

```diff
@@ -28,7 +28,6 @@ export function categoryReducer(state = initialState, action: DaffCategoryAction
     // applied by Magento, then this will result in a bug. Until Magento returns applied filters with a category call, this is
     // unavoidable.
     case DaffCategoryActionTypes.CategoryLoadAction:
-			if(!action.request.applied_filters) action.request.applied_filters = [];
```

Comment:
Are loads assumed to pass filters now? I'm not certain I see why this should be removed.